src/Composer/PatchAddCommand.php

```diff
@@ -44,6 +48,7 @@ protected function execute(InputInterface $input, OutputInterface $output) {
     $package = $input->getArgument('package');
     $description = $input->getArgument('description');
     $url = $input->getArgument('url');
+    $updateDevMode = !$input->getOption('update-no-dev');
```

Comment:
Maybe should conform to the standard `--no-dev` as the other composer standard commands?

Reply:
You're right. I just saw, that I was following what Composer is doing on their RequireCommand instead of the UpdateCommand. This should be adjusted.